Tmain/interactive-mode.d/test.c

```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+
+int main(int argc, char **argv) {
+  say_hello();
+}
+
+void say_hello() {
+  printf("hello world\n");
+}
```

Comment:
better to have valid test source files, so put the `say_hello()` declaration after its first use (e.g. above `main()`)